soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -598,7 +598,7 @@ namespace Settings {
   };
 
   Menu startingItems            = Menu::SubMenu("Items",                &startingItemsOptions, false);
-  Menu startingSongs            = Menu::SubMenu("Ocarina Songs",        &startingSongsOptions, false);
+  Menu startingSongs            = Menu::SubMenu("Starting Songs",       &startingSongsOptions, true);
```

Comment:
This menu isn't necessarily for us, but rather for 3d rando to display the options available for the player. (You already exposed the options through our ImGui implementation.)
To answer your question in the OP, you're printing twice since the `true` here refers whether to print to the spoiler log.

Reply:
Yeah, I realize that, but without using their method, it either writes all the songs unorganized in the spoiler log, or I have to change all the names to "start with 'song name'" Which, either way is fine, but I'm just not sure what would be preferred. Using their sub menu makes it cleaner everywhere, minus the fact that it results in a double entry

Reply:
Personally, I would find it consistent with all the other options if starting songs began with "Start With". I wonder if doing this and removing the loop I mentioned in this review will a) have a single set of song options and b) organize the result.

Reply:
I did it before, and if I remember correctly it did organize it in the spoiler log. not sure what you mean by single set of song options though

Reply:
You mentioned a double entry somewhere? Just wanted to make sure we were on the same page.

Reply:
Oh yes, my bad. Yeah, it'll definitely do both those things.

Reply:
![image](https://user-images.githubusercontent.com/115201185/209411143-4cbb3bb6-03fe-4e66-a063-24832caf5a75.png)

That's what the menu looks like now with the changes and only one entry in the spoiler log for each song.